.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
- [ ] [OPTIONAL] Please make sure you don't have unnecessary files in your project director. This makes the project dirty. You can add `.vscode `to `.gitignore`.

Reply:
thanks